test/test.js

```diff
@@ -28,11 +29,59 @@ describe('BirthdayReminder', () => {
     });
 
     describe('#run()', () => {
-        it('sends not a signal when it is not the birthday date', async function () {
-            const simulatedDate = getConfigForDate(new Date(2020, 11, 21));
+        it('the current date is far from the birthday date', async function () {
+            const simulatedDate = getConfigForDate(new Date(2019, 05, 21));
             let app = await buildApp(simulatedDate);
             return app.run().then((signal) => {
-                assert.equal(signal, null);
+                assert.ok(signal);
+                assert(signal.message.includes(simulatedDate.applet.user.nameOfTheBirthdayPerson));// on verifie si on recoit le signal avec le nom dedans
```

Comment:
Please do not comment in French. Don't forget that this will be public. And everyone can see